circle.yml

```
@@ -50,9 +50,9 @@ jobs:
               git config --global user.email "release@hyperanna.com"
               git config --global user.name "Release"
               git branch -u origin/${CIRCLE_BRANCH}
-              sbt "release release-version 1.${CIRCLE_BUILD_NUM} with-defaults skip-tests"
+              git tag -a v1.${CIRCLE_BUILD_NUM} -m "Version 1.${CIRCLE_BUILD_NUM}"
```

I wouldn't bother with an annotated tag - everyone else just does a lightweight tag. 

```suggestion
              git tag v1.${CIRCLE_BUILD_NUM}
```